openedx/core/djangoapps/catalog/management/commands/tests/test_cache_programs.py

```diff
@@ -170,3 +176,176 @@ def test_handle_missing_programs(self):
 
         for key, program in cached_programs.items():
             self.assertEqual(program, partial_programs[key])
+
+
+@skip_unless_lms
+@httpretty.activate
+class TestMultitenantCachePrograms(CatalogIntegrationMixin, CacheIsolationTestCase, SiteMixin):
```

Comment:
Please do not just copy all of the existing tests. Not only is it hard to tell what you're changing when you do this, the existing test class `TestCachePrograms` is all we want left when the Waffle switches are removed. Either add test cases to that class that cover just what you need, or modify the existing ones.